src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -294,6 +294,10 @@ const PaymentFormContent: React.FC<PaymentFormContentProps> = ({
       setupFutureUsage: null,
       mode: "setup",
       payment_method_types: [paymentType],
+      // @ts-ignore Stripe type issue
+      paymentMethodOptions: {
+        us_bank_account: { verification_method: "instant" },
```

Comment:
This option will be applied when `sepa_debit` is selected too. Seems to be ok?

Also, the typing would be helpful as now we have different casing for `payment_method_types` 🐍  and `paymentMethodOptions` 🐫 . 😝